### PR TITLE
Issue #15: Amigo Requests Page

### PR DESCRIPTION
**Pull request by mckoon**
Issue #15: Amigo Requests Page

Creates the Amigo Requests Page.
Creates amigo-share-request-card component.
Creates AmigoShareRequest object.

### CODE REVIEW

File: app/amigo-share-request-card/amigo-share-request-card.component.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+            <button secondary clear small>
+                <ion-icon name="remove-circle"></ion-icon>
+                <div>Decline</div>
+            </button>
+
+            <button primary clear small>
+                <ion-icon name="add-circle"></ion-icon>
+                <div>Accept</div>
+            </button>
```
Comment (mckoon) [inline app/amigo-share-request-card/amigo-share-request-card.component.html]: These buttons don't actually do anything yet.

They will eventually set `accepted` to `true` or `false` and then call a `save` method in the `AmigoShareRequestService` that will POST it to the server.

Just getting the page layout, components, and views in place in this PR.

**Comment by mckoon** [comment]
amigorequests
sidenav

**Comment by geoffreygraddy** [comment]
👍

**Comment by jleopold28** [comment]
👍

**Comment by mckoon** [comment]
merging

**Pull request merged**